cmd/root.go

```diff
@@ -38,9 +37,7 @@ func IsManagementAPI(cmd *cobra.Command) bool {
 
 func PromptLogin(ctx context.Context, fsys afero.Fs) error {
 	if _, err := utils.LoadAccessTokenFS(fsys); err == utils.ErrMissingToken {
-		return login.Run(ctx, os.Stdout, login.RunParams{
-			Fsys: fsys,
-		})
+		return errors.New("You need to be logged-in in order to use Management API commands. Run `supabase login` first.")
```

Comment:
```suggestion
		utils.CmdSuggestion = fmt.Sprintf("Run %s first.", utils.Aqua("supabase login"))
		return errors.New("You need to be logged-in in order to use Management API commands.")
```

Just a nitpick for consistency of error message style.